test/tests/test_environments/test_go_env.sh

```diff
@@ -70,7 +70,7 @@ fission fn update --name $fn_poolmgr --pkg $pkgName
 fission fn update --name $fn_nd --pkg $pkgName
 
 log "Waiting for router & pools to catch up"
-sleep 5
```

Comment:
300 is quite a lot, did we try with smaller numbers?

Reply:
Done